tests/e2e/features/3DViewer.feature

```diff
@@ -0,0 +1,43 @@
+Feature: preview 3D model
+  As a user
+  I want to preview a 3D model
+  So that I can make sure this is the correct file (e.g. before sharing it with others)
+
+  Background:
+    Given the user has logged in with username "admin" and password "admin"
+    And a 3D model (.glb file) has been uploaded
+    #should the filename be specified here?
```

Comment:
we can do it with data table. with this, we can add more model types easily in the future
```suggestion
    And the user has uploaded the following 3D models:
      | mode1.glb |
    #should the filename be specified here?
```